api/index.js

```diff
@@ -4,8 +4,23 @@ const express = require('express');
 const http = require('http');
 const app = express();
 const server = http.createServer(app);
+const socket_io = require('socket.io');
 
-const config = require('./config');
+//const config = require('./config');
+
+server.listen(8080);
+var io = socket_io();
+
+io.attach(server);
```

Comment:
Can you make a separate file to put all the socket endpoints at?

Reply:
Sorry, but what are the endpoints again? Like what would the file look like/do?